components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -122,10 +122,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ================================================================== -->
 <!-- FATES default parameter file                                       -->
 <!-- ================================================================== -->
-<fates_paramfile >lnd/clm2/paramdata/fates_params_default_12pft_api16.c210527.nc</fates_paramfile>
+<fates_paramfile >lnd/clm2/paramdata/fates_params_api.16.1.0_12pft_c210630.nc</fates_paramfile>
```

Comment:
FYI @glemiex, here is the new file: 
https://web.lcrc.anl.gov/public/e3sm/inputdata/lnd/clm2/paramdata/fates_params_12pft_api17.c211116.nc

Reply:
Thanks @rgknox